packages/core/core/src/services/document-service/utils/populate.ts

```diff
@@ -22,12 +22,7 @@ export const getDeepPopulate = (uid: UID.Schema, opts: Options = {}) => {
           break;
         }
 
-        // TODO: Should this just be a plain list?
-        // Ignore createdBy, updatedBy, ...
-        const isVisible = contentTypes.isVisibleAttribute(model, attributeName);
```

Comment:
So with the populateCreatorFields I think you can keep the isVisible, otherwise it's looses it's purpose 🤔

Reply:
could you elaborate a bit more? 🤔 doesn't populateCreatorFields has to be set on the content type level? 
This bit ensures the creator fields are populated for all content types, regardless of that config

Reply:
So populateCreatorFields defines if they are "allowed to be populated" what it does is set them to private or not but it doesn't actually populate them if you don't request them to avoid loading stuff when you do'nt want to see it. the name of the option could be better :D

Reply:
It seems a bit odd to me, that to have the creator fields on the published data, you need to activate a "populateCreatorFields" in your content type. Shouldn't we always have that data available?

Maybe we could keep the visible attribute check, but always include the creator fields, wdyt?

Reply:
I think there is a confusion. 

the populateCreatorFields just trigger private/not private on the fields. This is done to avoid leaking user information so devs have to opt-in to allow fetching creator fields.

You definitly can remove the isVisibleAttribute here as the purpose is to hide things in the admin ui not in the API response :)

Reply:
Ah I see :) thank you! You were saying that in that scenario, the isVisible check lost its purpose 🤦‍♂️  mb

I gave it another thought, and there are specific fields we might not want to copy here, such as the review workflow assignee or stage. I might add those as excluded , without the isVisibleAttribute field.

Reply:
true, you have 2 options

- Specificaly populate creator fields & exclude visible
- Specifically exclude some visible and select all 

Sounds like 1. is the safer approach 🤔